tests/20-verify.js

```diff
@@ -142,16 +44,6 @@ describe('eddsa-2022 (verify)', function() {
             const credential = credentials.clone('issuedVc');
             await verificationSuccess({credential, verifier});
           });
-        it('If the "proofValue" field is not a multibase-encoded base58-btc ' +
-          'value, an INVALID_PROOF_VALUE error MUST be returned.',
-        async function() {
-          this.test.cell = {columnId, rowId: this.test.title};
-          const credential = credentials.clone('issuedVc');
-          // remove the initial Z
-          credential.proof.proofValue = credential.proof.proofValue.slice(
-            1, credential.proof.proofValue.length);
-          await verificationFail({credential, verifier});
-        });
         it('If the "proofValue" field, when decoded to raw bytes, is not ' +
```

Comment:
Is this a general test for `proofValue` values? Can this be moved into `data-integrity-test-suite-assertion` lib?